docs/sources/clients/promtail/configuration.md

```diff
@@ -1797,7 +1797,7 @@ server:
 positions:
   filename: /var/log/positions.yaml # This location needs to be writeable by Promtail.
 
-client:
+clients:
```

Comment:
Q: if we are chaning `client` -> `clients` then below `url` should also be changed to array?

Reply:
yes, thanks for catching this